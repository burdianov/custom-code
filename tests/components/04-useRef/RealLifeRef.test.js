import { shallow } from 'enzyme';
import RealLifeRef from './../../../components/04-useRef/RealLifeRef';

describe('pruebas en <RealLifeRef />', () => {
  const wrapper = shallow(<RealLifeRef />);

  test('debe mostrarse correctamente', () => {
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('MultipleCustomHooks').exists()).toBe(false);
  });

  test('debe de mostrar el componente <MultipleCustomHooks />', () => {
    const button = wrapper.find('button');

    button.simulate('click');

    expect(wrapper.find('MultipleCustomHooks').exists()).toBe(true);
  });
});
