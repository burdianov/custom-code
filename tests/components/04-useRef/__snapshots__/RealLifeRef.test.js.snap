// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <RealLifeRef /> debe mostrarse correctamente 1`] = `
<Fragment>
  <h1>
    Real Life Ref
  </h1>
  <hr />
  <br />
  <button
    className="btn btn-primary mt-5"
    onClick={[Function]}
  >
    Show/Hide
  </button>
</Fragment>
`;
